e2e/wdio/driver/wdio.ts

```diff
@@ -61,7 +61,21 @@ export function goBack(): void {
     browser.back();
 }
 
-export function refreshPage(): void {
+export function refreshPage(isFullRefresh = false): void {
+    try {
+        browser.dismissAlert();
+    } catch {}
+    if (!isFullRefresh) {
+        const url = browser.getUrl();
+        click(`#toolbar-home-btn`);
+        if (browser.getUrl().includes(`/home`)) {
+            goBack();
+            return;
+        } else {
+            // failed to navigate, reset the url and perform full page refresh
+            browser.url(url);
```

Comment:
If we're in this branch we will be refreshing page 2 times. Here & `browser.refresh();` later.

Reply:
Nice catch. I was just handling a weird edge cases if any and didn't notice `browser.url` performs normal page redirects. Fixed